pom.xml

```diff
@@ -154,28 +199,127 @@
                 <dependency>
                     <groupId>org.springframework.cloud</groupId>
                     <artifactId>spring-cloud-gcp-starter</artifactId>
+                    <version>${spring.cloud.version}</version>
                 </dependency>
                 <dependency>
                     <groupId>org.springframework.cloud</groupId>
                     <artifactId>spring-cloud-gcp-starter-sql-mysql</artifactId>
-                    <version>1.1.0.RELEASE</version>
+                    <version>${spring.cloud.version}</version>
                 </dependency>
             </dependencies>
         </profile>
     </profiles>
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
what is the reason this was added? I saw it in your config when we were working earlier but didn't ask